app/views/my_facilities/_registrations_details_v2.html.erb

```diff
@@ -0,0 +1,106 @@
+              </th>
+              <th class="row-label sort-label sort-label-small" data-sort-method="number" style="width: 120px;">
+                Total registered<br>patients
+              </th>
+              <th class="row-label sort-label sort-label-small" colspan="2" style="width: 80px;">
+                6-month change
+              </th>
+              <% (@start_period..@period).each do |period| %>
+                <th class="row-label sort-label sort-label-small" data-sort-method="number" colspan="2">
+                  <%= period %>
+                </th>
+              <% end %>
+            </tr>
+          </thead>
+          <tbody>
+            <% @facilities_by_size[size].each do |facility| %>
+              <% six_month_rate_change = ((@data_for_facility[facility.name]["registrations"][@period] - @data_for_facility[facility.name]["registrations"][@start_period])/facility.opd_load.to_f) * 100.0 %>
+              <tr data-row="<%= facility.slug %>" data-trend-color="<% if six_month_rate_change > 0 %>green<% elsif six_month_rate_change < 0 %>red<% else %>grey<% end %>">
+                <td class="type-title">
+                  <%= link_to(reports_region_path(facility, report_scope: "facility")) do %>
+                    <%= facility.name %>
+                  <% end %>
+                </td>
+                <td>
+                  <%= number_or_zero_with_delimiter(@data_for_facility[facility.name]["adjusted_registrations"].values.last) %>
+                </td>
+                <td>
+                  <%= number_or_zero_with_delimiter(@data_for_facility[facility.name]["cumulative_registrations"].values.last) %>
+                </td>
+                <td>
+                  <div class="w-80px h-20px">
+                    <canvas id=<%= facility.slug %>></canvas>
+                  </div>
+                </td>
+                <td class="fw-bold ta-center <% if six_month_rate_change > 0 %>c-green-dark<% elsif six_month_rate_change < 0 %>c-red-dark<% else %>c-grey-dark<% end %>"> 
+                  <%= number_to_percentage_with_symbol(six_month_rate_change, precision: 0) %>
+                </td>
+                <% (@start_period..@period).each do |period| %>
+                  <td>
+                    <%= @data_for_facility[facility.name]["registrations"][period] %>
+                  </td>
+                  <td class="type-percent">
+                    <% registrations_rate = @data_for_facility[facility.name]["registrations"][period] / facility.opd_load.to_f * 100.0 %>
+                    <em data-rate="<%= registrations_rate %>">
+                      <%= number_to_percentage(registrations_rate || 0, precision: 0) %>
+                    </em>
+                  </td>
+                <% end %>
+              </tr>
+            <% end %>
+          </tbody>
+        </table>
+      <% end %>
+    </div>
+  </div>
+</div>
+<script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
```

Comment:
we should get this into our asset bundle properly at some point.  Not necessary for this PR.